source/tests/llassetgen-tests/Atlas.cpp

```diff
@@ -0,0 +1,37 @@
+#include <gmock/gmock.h>
+#include <llassetgen/Atlas.h>
+
+using namespace llassetgen;
+
+std::string atlasTestDestinationPath = "../../";
+std::vector<Vec2<size_t>> atlasTestSizes {{1, 1}, {34, 5}, {23, 79}, {16, 70}, {91, 64}, {98, 82}, {54, 63}, {100, 6}};
+
+TEST(AtlasTest, CreateDistanceFieldAtlas) {
+    std::vector<Image> glyphs;
+    glyphs.reserve(atlasTestSizes.size());
+
+    for (const auto& size : atlasTestSizes) {
+        glyphs.emplace_back(size.x, size.y, 1);
+    }
+
+    Packing p = shelfPackAtlas(atlasTestSizes.begin(), atlasTestSizes.end(), false);
+    Image atlas = distanceFieldAtlas<DeadReckoning>(glyphs.begin(), glyphs.end(), p);
```

Comment:
For completeness we should probably factor this test out and call it with both algorithms, but it is probably not that important.